pkgs/tools/security/sha1collisiondetection/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, libtool }:
+
+stdenv.mkDerivation  rec {
+  pname = "sha1collisiondetection";
+  version = "git-20170221";
```

Comment:
Can you elaborate on the benefit of replacing the standard utility? sha1 has been known to be unsuitable for certain applications for a long time, our energy is better spent coordinating to phase it out where it makes sense to do so (as with md5), imo.